src/transformers/models/align/processing_align.py

```diff
@@ -82,15 +119,28 @@ def __call__(self, text=None, images=None, padding="max_length", max_length=64,
             - **pixel_values** -- Pixel values to be fed to a model. Returned when `images` is not `None`.
         """
         if text is None and images is None:
-            raise ValueError("You have to specify either text or images. Both cannot be none.")
+            raise ValueError("You must specify either text or images.")
 
         if text is not None:
-            encoding = self.tokenizer(
-                text, padding=padding, max_length=max_length, return_tensors=return_tensors, **kwargs
-            )
+            text_kwargs = {
+                **self.processing_kwargs["text_kwargs"],
+                **self.processing_kwargs["common_kwargs"],
+                **kwargs,
+            }
+            encoding = self.tokenizer(text, **text_kwargs)
 
         if images is not None:
-            image_features = self.image_processor(images, return_tensors=return_tensors, **kwargs)
+            images_kwargs = {
+                **self.processing_kwargs["images_kwargs"],
+                **self.processing_kwargs["common_kwargs"],
+                **kwargs,
+            }
+            image_features = self.image_processor(images, **images_kwargs)
+
+        # BC for explicit return_tensors
+        common_kwargs = {**self.processing_kwargs["common_kwargs"], **kwargs}
```

Comment:
Why do we prepare common_kwargs here and not use them? Is it for consistency in the processor patterns?

Reply:
will correct this part in `align` :)